api/odigos/actions/v1alpha1/deleteattribute_types.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/keyval-dev/odigos/common"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DeleteAttributeSpec defines the desired state of DeleteAttribute action
+type DeleteAttributeSpec struct {
+	ActionName string                       `json:"actionName,omitempty"`
+	Notes      string                       `json:"notes,omitempty"`
+	Disabled   bool                         `json:"disabled,omitempty"`
+	Signals    []common.ObservabilitySignal `json:"signals"`
```

Comment:
We can extract these to a shared struct and embed it in every action.
Not urgent for this PR